STAR.Domain/Contractor.cs

```diff
@@ -10,6 +10,16 @@ public class Contractor {
         public string FirstName { get; set; }
         [DisplayName("Last Name")]
         public string LastName { get; set; }
+        //[DisplayName("Primary Phone")]
```

Comment:
I actually put that there as a placeholder for when we add new fields to the contractor. I had added it to the db etc. but then realized I was out of scope.